### DISCUSSION TITLE
Error 404 not found!!! please please help me!!

### DISCUSSION THREAD

**Discussion by JACC-THAKUR** [vercel/community#5301] [Help]
### Summary

not able to see the deployed website ,showing error 404 not found what to do for it ,how can i see correctly my deployed project please please help me !!!!!!!  please someone tell me the complete process step by step  for deploying the website and also how can i sort out that error 404 not found 

### Example

https://vercel.com/shubham-thakurs-projects/new-inner/6Pp6vdS8dxaVhLABceQyUKoGE94h

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/122075676/4fcea9e5-8446-4fd9-9c51-beaaa9adf62b)

**Comment by krishnaacharyaa** [comment]
What is the stack are you using, is it like a simple html css website? Can you share the project link here

**Comment by HRSAndrabi** [comment]
The answer depends on which stack you are using, but it might be helpful to have a look at [relevant documentation](https://vercel.com/docs/frameworks/nextjs?utm_source=next-site&utm_medium=docs&utm_campaign=next-website).
